File: C#/uri-1002.cs
// URI - 1002

using System;

namespace URI
{
    class Program
    {
        static void Main(string[] args)
        {
            double radius = double.Parse(Console.ReadLine());

            Console.WriteLine("A=" + (3.14159 * Math.Pow(radius, 2)).ToString("F4"));
        }
    }
}

File: Node/uri-1002.js
// URI - 1002

const readline = require('readline')

const rl = readline.createInterface({
    input: process.stdin,
    output: process.stdout
})

rl.on('line', input => {
    const radius = parseFloat(input)

    console.log('A=' + (3.14159 * Math.pow(radius, 2)).toFixed(4))

    rl.close()
})